metricbeat/module/openmetrics/collector/_meta/samelabeltestdata/config.yml

```diff
@@ -1,8 +0,0 @@
-type: http
```

Comment:
I don't think this directory should be removed. It's an actual test case that had been working so far, right?
Unless there is strong reason for it, we should not remove it.

Reply:
I will fix that. I still think one of the 2 is not being used, but maybe that is a change for a different PR.